content/00.front-matter.md

```diff
@@ -1,4 +1,3 @@
-# Manubot Rootstock: Manuscript Title
```

Comment:
The title is being added through the metadata. This is part of the default functionality of the ``yaml_metadata_block`` markdown extension.